frontend/src/main/scala/akka/viz/frontend/components/components.scala

```diff
@@ -1,7 +1,7 @@
 package akka.viz.frontend.components
 
 import akka.viz.frontend.FrontendUtil._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004071389/issues/source?bid=3121282&fileBranchId=3122179#l3)